.travis.yml

```diff
@@ -3,7 +3,15 @@ language: node_js
 node_js:
   - "6"
 
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
This was copied from ember-cookies (https://github.com/simplabs/ember-cookies/blob/master/.travis.yml#L7). Chrome installation fails without it.